e2e/src/DappList.spec.js

```diff
@@ -1,6 +1,73 @@
 import DappListRecent from './usecases/DappListRecent'
-import { quickOnboarding } from './utils/utils'
+import DappListDisplay from './usecases/DappListDisplay'
+import { quickOnboarding, waitForElementId } from './utils/utils'
 import { launchApp } from './utils/retries'
+import fetch from 'node-fetch'
+
+/**
+ * Fetch the Dapp list from the cloud function using node-fetch
+ * @param {string} userAgent: user agent to use for dapp list fetch
+ * @returns {object|null} dappList: list of dapps
+ */
+export async function fetchDappList(userAgent = '') {
+  let dappList = null
+  try {
+    const response = await fetch(
+      'https://us-central1-celo-mobile-alfajores.cloudfunctions.net/dappList',
+      {
+        headers: {
+          'User-Agent': userAgent,
+        },
+      }
+    )
+    if (response.status === 200) {
+      dappList = await response.json()
+    }
+  } catch {}
+  return dappList
+}
+
+/**
+ * From the home screen, navigate to the dapp explorer screen
+ * Disable detox device synchronization to allow for dapp list to load
+ * https://github.com/wix/Detox/issues/2799
+ */
+export async function navigateToDappList() {
```

Comment:
these functions are in the dappList utils file now right? they can be removed here?